server.py

```diff
@@ -12,6 +12,7 @@
 
 IDENTITY = 'voice_test'
 CALLER_ID = 'quick_start'
+CALLER_PHONE = '+1****'
```

Comment:
We should also specify this as an environment variable like we do with the other values here so we can have the Heroku deployment still work easily.